python/src/iceberg/table/partitioning.py

```diff
@@ -24,7 +24,7 @@ class PartitionField:
 
     Attributes:
         source_id(int): The source column id of table's schema
-        field_id(int): The partition field id across all the table metadata's partition specs
+        field_id(int): The partition field id across all the table partition metadata specs
```

Comment:
nit: I think [partition spec](https://iceberg.apache.org/spec/#partition-specs) here refer to a single entity in iceberg jargon, so I feel `partition metadata specs` does not confer the same meaning

Reply:
I've updated it to `The partition field id across all the table partition specs metadata` WDYT?